packages/alfa-dom/src/get-attribute.ts

```diff
@@ -17,43 +39,137 @@ const attributeMaps: WeakMap<Element, Map<string, string>> = new WeakMap();
 export function getAttribute(
   element: Element,
   name: string,
-  options: Readonly<{
-    trim?: boolean;
-    lowerCase?: boolean;
-  }> = {}
+  selectorNamespace?: Namespace | string | null | AttributeOptions,
+  options: AttributeOptions = {}
 ): string | null {
-  let attributeMap = attributeMaps.get(element);
+  if (selectorNamespace !== null && typeof selectorNamespace === "object") {
```

Comment:
Is there an easier/prettier way of figuring out whether or not it is of an AttributeOptions interface?